etn-ccis/blui-react-workflows
# added remember me functionality to login screen

Fixes # added remember me functionality to login screen.



#### Changes proposed in this Pull Request:

-
-
-



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- yarn start:example2 or yarn start:example3
- Go to login page url: http://localhost:3000/login
- Enter email, password and select remember me checkbox then click "log in" button.
- In the homepage, click logout from the top right corner user menu option.
- Now check your login page that email and remember me pre-fill values.



#### Any specific feedback you are looking for?

-
